packages/build/config/tsconfig.common.json

```diff
@@ -7,7 +7,6 @@
     "skipLibCheck": true,
     "strict": true,
 
-    // FIXME(bajtos) LB4 is not compatible with this setting yet
```

Comment:
TypeScript tools such as `tsc` does allow comments in the json file.

Reply:
See https://github.com/microsoft/TypeScript/pull/5450.

VS Code editor coined the term "JSON with comments", a.k.a. "jsonc" - see https://code.visualstudio.com/Docs/languages/json#_json-with-comments

Microsoft also provides a Node.js implementation for parsing "JSON with comments", see https://github.com/Microsoft/node-jsonc-parser